infra/auth-lookup/main.tf

```diff
@@ -0,0 +1,39 @@
+data "aws_caller_identity" "current" {}
+
+locals {
+  common_tags = "${map(
+    "managedBy" , "terraform",
+    "Name"      , "${var.DSS_INFRA_TAG_SERVICE}-acldynamodb",
+    "project"   , var.DSS_INFRA_TAG_PROJECT,
+    "env"       , var.DSS_DEPLOYMENT_STAGE,
+    "service"   , var.DSS_INFRA_TAG_SERVICE,
+    "owner"     , var.DSS_INFRA_TAG_OWNER
```

Comment:
Questioning whether the `owner` name here is the right name - since that implies some mechanism during creation events that creates a record indicating which uuid is owned by which user. Should this be `users` containing a list of allowed users, just like `groups` is a list of allowed groups?

Reply:
(Especially considering `update` events are only checking whether user has both read and create access)

Reply:
Im a little confused.....
the use of `owner` here (ln.10) is to ensure that infra is labeled correctly during deployment, it allows resources to be grouped when using the cost explorer.

there is also an owner attribute on ln.28 which is part of the actual schema.... 
we can drop the idea of the owner, and actually that might be preferable as its difficult to keep track of, a list of 'users' might be better....

Reply:
Okay - I think I misunderstood and confused the `owner` on line 10 (which is for AWS and tagging the dynamodb table) with the `owner` attribute in the table.